src/background/user_actions.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * 🎶 No alarms and no surprises 🎶
+ * 🎶 No alarms and no surprises 🎶
+ * https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/User_actions
+ */
+
+import * as excmds from "@src/.excmds_background.generated"
+import * as R from "ramda"
+import { messageTab } from "@src/lib/messaging"
+
+export function escapehatch() {
+    browser.sidebarAction.open()
+    browser.sidebarAction.close()
```

Comment:
In theory this is racy as they both return promises. In practice, even with `cpulimit --limit 50` it seems to work fine.